build.sbt

```diff
@@ -1074,6 +1074,7 @@ lazy val `polyglot-api` = project
 
 lazy val `language-server` = (project in file("engine/language-server"))
   .settings(
+    commands += WithDebugCommand.withDebug,
```

Comment:
This is useful for debugging tests in `language-server` project with `withDebug` command.